librosa/display.py

```diff
@@ -989,9 +994,15 @@ def __coord_cqt_hz(n, fmin=None, bins_per_octave=12, **_kwargs):
     fmin = fmin * 2.0**(_kwargs.get('tuning', 0.0) / bins_per_octave)
 
     # we drop by half a bin so that CQT bins are centered vertically
-    return core.cqt_frequencies(n+1,
-                                fmin=fmin / 2.0**(0.5/bins_per_octave),
-                                bins_per_octave=bins_per_octave)
+    freqs = core.cqt_frequencies(n+1,
+                                 fmin=fmin / 2.0**(0.5/bins_per_octave),
+                                 bins_per_octave=bins_per_octave)
+
+    if np.any(freqs > 0.5 * sr):
+        warnings.warn('Frequency axis exceeds Nyquist. '
```

Comment:
the wording is a little bit quirky, but i don't have anything better to propose.
time will tell

Reply:
Agreed.  I'm thinking the suggestion in the warning should be sufficient to poke the user into looking in the right place, but couldn't find a more eloquent way to word it.